react-hot-button/my-app/src/App.tsx

```diff
@@ -0,0 +1,28 @@
+import { useState } from 'react';
+import { HotButton } from './hotButton';
+import './App.css';
+import './hotButton.css';
```

Comment:
You would usually import this in the component file that depends on it, hotButton.tsx. (And, as mentioned below, it should be named HotButton.css.)